lib/world_database.rb

```diff
@@ -1,3 +1,4 @@
+require "sqlite3"
 
```

Comment:
@boveus Hey man, noticed that we were requiring sqlite3 in our test helper folder, moved it into the actual class that was creating databases.